src/RefocusCollectorEval.js

```diff
@@ -98,14 +97,16 @@ class RefocusCollectorEval {
    * @param {String} status - Status code.
    */
   static statusCodeMatch(tr, status) {
+    let func;
     if (tr.errorHandlers) {
       Object.keys(tr.errorHandlers).forEach((statusMatcher) => {
         const re = new RegExp(statusMatcher);
         if (re.test(status)) {
-          return tr.errorHandlers[statusMatcher];
+          func = tr.errorHandlers[statusMatcher];
```

Comment:
The way I'm reading this, the change here is basically "return the last matching handler" instead of "return the first matching handler" ... what's the rationale for this? Tests for multiple matches? What is the "right" way to handle multiple matches? (It needs to be predictable and explainable to SGT devs.)

Reply:
Actually, this is exactly what we have in the refocus-collector repo https://github.com/salesforce/refocus-collector/blob/ae007cbfe735a90e1248b01cad1bd848ff7f485b/src/remoteCollection/handleCollectResponse.js#L142

We were returning the last matching handler.

Reply:
Right, but the comment there says otherwise "Use the first one to match", I am not sure what we want here.

Reply:
I think during the build we should validate and reject if any key matches more than one status, for every status [100-599].

Reply:
Also, when I think about it, there cannot be multiple matches unless the user has added duplicate keys to the `errorHandlers` attribute, correct?

Reply:
A user could inadvertently be setting up multiple matches if they don't understand how regex works

Reply:
Ok, I will add test for multiple matches and update the comment saying that the last matching handler will be returned.